quantecon_book_networks/data.py

```diff
@@ -104,14 +104,15 @@ def introduction():
     ch_data = {}
 
     ## Crude oil
-    data_file = "data/crude_oil_sitcr2_3330_yr2019/data.csv"
+    data_file = "data/crude_oil_sitcr2_3330_yr2021/data.csv"
     data_file = pkg_resources.resource_stream(__name__, data_file)
     crude_oil = pd.read_csv(data_file, dtype={'product_id': str})
 
     exporters = crude_oil.groupby(by=["location_code"]).sum().sort_values("export_value", ascending=False)[:10].index
 
     importers = crude_oil.groupby(by=["partner_code"]).sum().sort_values("export_value", ascending=False)[:21].index
-    importers = set(importers.drop("ANS"))
+    # importers = set(importers.drop("ANS"))
```

Comment:
@HengchengZhang has `ANS` been dropped from the dataset?

Reply:
ANS was in the data prior but it is not a valid ISO3C code

Reply:
Yes. Since we are using a different dataset, there is no Alaska North Slope this time.

Shall I remove this code now?

Reply:
Yes please @HengchengZhang good to remove it if it isn't necessary